src/System Application/App/User Details/src/UserDetails.Codeunit.al

```diff
@@ -0,0 +1,35 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Security.User;
+
+/// <summary>
+/// Contains functionality related to retrieving user details.
+/// </summary>
+codeunit 774 "User Details"
+{
+    Access = Public;
+
+    /// <summary>
+    /// Retrieves the details of a user.
+    /// </summary>
+    /// <param name="UserDetails">The user details record to be populated.</param>
+    procedure GetUserDetails(var UserDetails: Record "User Details")
```

Comment:
This means you call UserDetails.GetUserDetails(UserDetails), maybe it should just be called Get? Then you would call UserDetails.Get(UserDetails); since the codeunit is already called UserDetails this seems a bit excessive to also name it that.